### DISCUSSION TITLE
How to make all components bigger?

### DISCUSSION THREAD

**Discussion by vitalijalbu** [segmentio/evergreen#1534] [Q&A]
Hi, seems components are too small. I'd like to make them bigger at once, is it possible? Something like input height:44px - button also 44px.

**Comment by brandongregoryscott** [comment]
Hi @vitalijalbu, unfortunately I don't think there's an easy way to scale them all at once. I would recommend overriding the heights for each component in a custom theme. Take a look at our documentation site for more info: [Theming](https://evergreen.segment.com/introduction/theming)
